projects/addon-doc/src/components/example/example-get-tabs.pipe.ts

```diff
@@ -0,0 +1,11 @@
+import {Pipe, PipeTransform} from '@angular/core';
+import {TuiDocExampleProcessed} from '@taiga-ui/addon-doc';
+
+@Pipe({name: 'tuiDocExampleGetTabs'})
+export class TuiDocExampleGetTabsPipe implements PipeTransform {
+    public transform(content: TuiDocExampleProcessed, defaultTab: string): string[] {
```

Comment:
We do not write `public`. Also let's just spread the `keys`, no need for an extra `const`. It's pretty clear without it, I think.